core/field_image.js

```diff
@@ -43,6 +43,19 @@ goog.require('goog.userAgent');
  */
 Blockly.FieldImage = function(src, width, height, opt_alt) {
   this.sourceBlock_ = null;
+
+  // Replace any message references in the arguments.
+  src = Blockly.utils.replaceMessageReferences(src)
+  if (goog.isString(height)) {
+    height = Blockly.utils.replaceMessageReferences(height);
+  }
+  if (goog.isString(width)) {
```

Comment:
To allow references (acting as constants) in JSON, much like block color:
{
  "type": "field_image",
  "src": "%{BKY_OPENING_QUOTE_IMAGE}",
  "width": "%{BKY_QUOTE_IMAGE_SIZE}",
  "height": "%{BKY_QUOTE_IMAGE_SIZE}",
  "alt": "\u201C"
}

Alternative suggestions?
(corrected @param type)